homeassistant/components/philips_js/media_player.py

```diff
@@ -54,24 +57,22 @@
 
 CONF_ON_ACTION = "turn_on_action"
 
-DEFAULT_NAME = "Philips TV"
 DEFAULT_API_VERSION = "1"
-DEFAULT_SCAN_INTERVAL = 30
-
-DELAY_ACTION_DEFAULT = 2.0
-DELAY_ACTION_ON = 10.0
 
 PREFIX_SEPARATOR = ": "
 PREFIX_SOURCE = "Input"
 PREFIX_CHANNEL = "Channel"
 
-PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
-    {
-        vol.Required(CONF_HOST): cv.string,
-        vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
-        vol.Optional(CONF_API_VERSION, default=DEFAULT_API_VERSION): cv.string,
-        vol.Optional(CONF_ON_ACTION): cv.SCRIPT_SCHEMA,
-    }
+PLATFORM_SCHEMA = vol.All(
+    cv.deprecated(CONF_HOST),
```

Comment:
I assume the intent is to entirely retire the YAML eventually. Deprecate all the fields?